feature-group-definitions/custom-elements.yml

```diff
@@ -0,0 +1,14 @@
+# Tests don't allow the more specific https://html.spec.whatwg.org/multipage/custom-elements.html (yet)
+spec: https://html.spec.whatwg.org/multipage/
```

Comment:
Should this link to the #custom-elements anchor on that page?

Reply:
That would pass the lint at least. But I think we should change the lint to allow for links like https://html.spec.whatwg.org/multipage/custom-elements.html. The extra complexity is checking that those links actually work.